editors/vscode/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "even-better-toml",
   "displayName": "Even Better TOML",
   "description": "Fully-featured TOML support",
-  "version": "0.21.0",
+  "version": "0.20.0",
```

Comment:
I'm not familiar with VS Code, but is this expected to go down?

Reply:
Both `0.20.0` and `0.21.0` are not published actually, and I believe `0.21.0` is a typo.